racetrack_commons/racetrack_commons/plugin/core.py

```diff
@@ -56,6 +56,10 @@ def markdown_docs(self) -> str | None:
         """
         return None
 
+    def run_action(self, **kwargs) -> Any:
+        """Call a supplementary action of a plugin"""
+        return None
```

Comment:
This is interesting - what kind of supplementary action would be example here? Also, it's not mentioned in the changelog and documentation.

Reply:
Cool :)